arch/arm/soc/nxp_kinetis/k6x/Kconfig.soc

```diff
@@ -96,6 +96,14 @@ config K64_FLASH_CLOCK_DIVIDER
 	  This option specifies the divide value for the K64 flash clock from the
 	  system clock.
 
+config HAS_SYSMPU
+	bool "Enable MPU"
+	depends on CPU_HAS_MPU
+	select NXP_MPU
+	default n
+	help
+	  Enable MPU
+
```

Comment:
This should be one level higher, in arch/arm/soc/nxp_kinetis/Kconfig, because the sysmpu exists on other parts outside the k6x series.